frontend-web/src/features/call-for-coffee/coffee-call-state/coffee-call-state.tsx

```diff
@@ -1,18 +1,18 @@
 import { Accessor, Component, For, Show, splitProps } from "solid-js";
-import { CoffeeCall, CoffeeCallStatus } from "../backend-communication/call-aggregation";
+import type { CoffeeCall } from "../backend-communication/models";
 
 export const CoffeeCallState: Component<{ coffeeCall: CoffeeCall }> = (props) => {
-  const participantsAsList = () => Array.from(props.coffeeCall.participants.values()).join(',');
+  const participantsAsList = () => props.coffeeCall.participants.join(',');
```

Comment:
```suggestion
  const participantsAsList = () => props.coffeeCall.participants.join(', ');
```

Not relevant to the PR, and we might at some point render the participants differently, but it's a easy fix for now.